src/FooterStyles.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export const Box = styled.div`
+  padding: 80px 60px;
+  background: #131921;
+  position: bottom;
```

Comment:
not sure about property-value as `bottom`, assuming you intended to use either `absolute` or `fixed` property-value